tests/components/pilight/test_init.py

```diff
@@ -403,7 +403,8 @@ def test_call_rate_delay_throttle_enabled(self):
         for i in range(3):
             action(i)
 
-        assert runs == []
+        self.hass.block_till_done()
+        assert runs == [0]
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6304022)